### PR TITLE
refactor: code

### PR DESCRIPTION
**Pull request by evilebottnawi**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [x] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

- update webpack-default, 
- move `index.js` inside lib (let's do not mix configuration files in root with code)
- forgot `'use strict'` in some places

### Breaking Changes

No

### Additional Info

No

### CODE REVIEW

**Comment by evilebottnawi** [comment]
After this PR i will start rewriting logger on webpack logger

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/coverage
+/dist
+/node_modules
+/test/fixtures
```
Comment (hiroppy) [inline .eslintignore]: We need the line break?

File: .prettierignore [hunk 2]
```diff
@@ -1,3 +1,5 @@
-coverage
-node_modules
-CHANGELOG.md
+/coverage
+/dist
+/node_modules
+/test/fixtures
+CHANGELOG.md
```
Comment (hiroppy) [inline .prettierignore]: ditto

File: package.json [hunk 3]
```diff
@@ -7,25 +7,26 @@
...
+    "lint:js": "eslint --cache lib test",
+    "lint": "npm-run-all -l -p \"lint:**\"",
     "release": "standard-version",
     "security": "npm audit",
-    "test:only": "jest",
-    "test:coverage": "npm run test:only -- --coverage",
-    "test:watch": "npm run test:only --watch",
+    "test:only": "cross-env NODE_ENV=test jest",
+    "test:watch": "cross-env NODE_ENV=test jest --watch",
+    "test:coverage": "cross-env NODE_ENV=test jest --collectCoverageFrom=\"lib/**/*.js\" --coverage",
```
Comment (hiroppy) [inline package.json]: It is better to separate to jest.config.js?

Comment (evilebottnawi) [reply package.json]: I think it is more simple than using `jest.config.js`. Maybe we revisit this in future.

File: .eslintignore (hunk 1 above)
Comment (evilebottnawi) [reply .eslintignore]: fixed

File: .prettierignore (hunk 2 above)
Comment (evilebottnawi) [reply .prettierignore]: fixed

File: package.json (hunk 3 above)
Comment (hiroppy) [reply package.json]: ok

**Pull request merged**